### PR TITLE
[ADD] stock_label_print: new module to print labels form pickings and MO

### PR DESCRIPTION
**Pull request by agaldona**
@avanzosc/developers review when possible please.

### CODE REVIEW

File: stock_label_print/README.rst [hunk 1]
```diff
@@ -0,0 +1,22 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=================
+Stock Label Print
+=================
+
+New module to print labels from reception pickings and manufacturing orders.
+* When the picking or the OF is done, a new button is showed in the header.
```
Comment (oihane) [inline stock_label_print/README.rst]: shown

**Review by oihane — APPROVED** [review]
LGTM

File: stock_label_print/data/report_paperformat.xml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+            <field name="margin_top">10</field>
+            <field name="margin_bottom">10</field>
+            <field name="margin_left">5</field>
+            <field name="margin_right">5</field>
+            <field name="header_line" eval="False" />
+            <field name="header_spacing">1</field>
+            <field name="dpi">90</field>
+        </record>
+    </data>
+</openerp>
```
Comment (Daniel-CA) [inline stock_label_print/data/report_paperformat.xml]: Falta return

**Comment by Daniel-CA** [comment]
:+1:

**Pull request merged**